docs/learn/documentation/versioned/index.html

```diff
@@ -37,7 +37,54 @@ <h4>API</h4>
 -->
 </ul>
 
-<h4>Deployment</h4>
+
+<h4>API</h4>
+
+<ul class="documentation-list">
+  <li><a href="api/overview.html">Overview</a></li>
+</ul>
+
+<h4>Core</h4>
+
+<ul class="documentation-list">
```

Comment:
Is this a result of an incorrect merge? for eg: These links should not be on the new website structure

Reply:
done